cadl-tests/src/test/java/com/enums/extensible/ExtensibleClientTest.java

```diff
@@ -1,16 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.extensibleenums;
+package com.enums.extensible;
 
-import com.extensibleenums.models.DaysOfWeekExtensibleEnum;
+import com.enums.extensible.models.DaysOfWeekExtensibleEnum;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-class ExtensibleEnumsClientTest {
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
avoid import *

Reply:
Thanks, updated, it's strange because I already configured avoid import * in intellij. I will check..